src/ring/middleware/authorization.clj

```diff
@@ -0,0 +1,58 @@
+(ns ring.middleware.authorization
+  (:require [clojure.string :as str]
+            [ring.util.parsing :as parsing]))
+
+(def ^:private re-token68
+  (re-pattern "[A-Za-z0-9._~+/-]+=*"))
+
+(def ^:private re-auth-param
+  (re-pattern (str "(" parsing/re-token ")\\s*=\\s*(?:" parsing/re-value ")")))
+
+(defn- parse-auth-params
+  [auth-params]
```

Comment:
We can put the arguments on the same line to make the expression more concise without impacting readability:

```clojure
(defn- parse-auth-params [auth-params]
```